pkgs/top-level/python-packages.nix

```diff
@@ -6408,14 +6421,6 @@ in {
       };
     };
 
-  pytest_6_1 = self.pytest_6.overridePythonAttrs (oldAttrs: rec {
```

Comment:
I imagine this might break things upstream. I think this should be fine for us, though.